dashboard/src/components/AppView/ServicesTable/ServicesTable.tsx

```diff
@@ -38,10 +33,9 @@ class ServiceTable extends React.Component<IServiceTableProps> {
             </tr>
           </thead>
           <tbody>
-            {services.map(
-              s =>
-                s.item && <ServiceItem key={`services/${s.item.metadata.name}`} service={s.item} />,
-            )}
+            {serviceRefs.map(s => (
+              <ServiceItem key={`services/${s.namespace}/${s.name}`} serviceRef={s} />
```

Comment:
`key={`services/${s.namespace}/${s.name}`}` related to my comment above. Wouldn't be great if serviceRef (s) could have a method to get this?

Reply:
Opted to just use the full resource URL here rather than creating another specific key, it has all we need to be unique and it doesn't matter if this key is verbose.